src/ShortcutPresenter.kt

```diff
@@ -151,10 +151,13 @@ public class ShortcutPresenter() : Disposable {
         if (shortcutText.isEmpty() || shortcutText == shownShortcut) return fragments
 
         when {
-            keymap.getKind() == KeymapKind.WIN -> fragments.addText(shortcutText)
-            macKeyStokesFont != null && macKeyStokesFont!!.canDisplayUpTo(shortcutText) == -1 -> {
+            keymap.getKind() == KeymapKind.WIN -> {
```

Comment:
I guess here I'm wrong. I've just tried to remove the need for loading all these "macKeyStokesFont" etc. Just because on OSX it works.